ChatSecure/Classes/Controllers/XMPP/OTRXMPPRoomYapStorage.h

```diff
@@ -12,11 +12,15 @@
 #import "OTRIncomingMessage.h"
 #import "OTROutgoingMessage.h"
 
+@class OTRXMPPRoomOccupant;
+
 @interface OTRXMPPRoomYapStorage : NSObject <XMPPRoomStorage>
 
 @property (nonatomic, strong) YapDatabaseConnection *databaseConnection;
 
 - (instancetype)initWithDatabaseConnection:(YapDatabaseConnection *)databaseConnection;
 
+- (OTRXMPPRoomOccupant *)roomOccupantForJID:(NSString *)jid roomJID:(NSString *)roomJID accountId:(NSString *)accountId inTransaction:(YapDatabaseReadTransaction *)transaction;
```

Comment:
Add comment to this method warning not to call it within in an existing transaction.